### DISCUSSION TITLE
What is the collision frequency?

### DISCUSSION THREAD

**Discussion by 23navi** [faker-js/faker#229] [General]
What is the probability of having the same internet.email()?
Like I am using email as the primary key and I just want to know how often can I expect the collision(same email id), so that I can increase the percentage  count!

**Comment by Shinigami92** [comment]
Theoretically `fname x lname x providers`
But you can even pass your own firstname and lastname and provider arguments. So if you have an infinite amount of these, the result would also be infinity. And infinit possibilities would result in near to zero collisions :shrug:

**Reply by 23navi** [reply]
so what is the value of  fname x lname x providers? Like

**Reply by Shinigami92** [reply]
I assume it's different for every locale, just look into the default sets yourself :slightly_smiling_face:

**Reply by 23navi** [reply]
> I assume it's different for every locale, just look into the default sets yourself 🙂

Thanks
